accounts/abi/event_test.go

```diff
@@ -173,7 +173,6 @@ func TestEventTupleUnpack(t *testing.T) {
 	type EventTransferWithTag struct {
 		// this is valid because `value` is not exportable,
 		// so value is only unmarshalled into `Value1`.
-		value  *big.Int
 		Value1 *big.Int `abi:"value"`
```

Comment:
A whaa? Why is this weird Value1 thing here? Why not have `Value`?

Reply:
Oh, just read the comment and I should probably not remove this field because it's part of the test.